code/modules/mob/living/carbon/carbon.dm

```diff
@@ -14,7 +14,6 @@
 /mob/living/carbon/updatehealth()
 	..()
 	med_hud_set_health()
-	med_hud_set_status()

```

Comment:
moved to update_stat() (which updatehealth() calls). This is so med hud updates properly if a mob's stat changes without calling updatehealth().